backend/setup-without-docker.md

```diff
@@ -14,6 +16,8 @@ Update *project/server/config.py*, and then run:
 $ export APP_NAME="ALIAS CHECK API"
```

Comment:
`flask-smorest` needs `flask = "==2.1.2"` to run since one of its method was removed in 2.2
```
ImportError: cannot import name 'MethodViewType' from 'flask.views'
```

Reply:
this was the error i was telling you about when using the pipfile, which is why i resorted to using a requirements.txt which specifies the flask version as 2.1.2

Reply:
For some reason, I was able get it working with Pipfile